collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
I agree that `mapper-lens` is a good two-way generalization of the one-way traditional `map`, and that seems to be an excellent reason to name it `map-lens`. But which naming convention takes precedence? I don't know.